.changeset/new-rules-vanish.md

```diff
@@ -0,0 +1,5 @@
+---
+"@sveltejs/kit": patch
```

Comment:
Is it going to matter if these are double quotes instead of single quotes?

Reply:
this was created via the link in the comment — https://github.com/sveltejs/kit/pull/4510#issuecomment-1087947182 — so i certainly hope not

Reply:
Huh, weird that the CLI does something different than the bot. Anyway, if this wasn't the result of you manually writing the file, I'm not worried.